Mopy/bash/game/skyrim/constants.py

```diff
@@ -17117,7 +17117,8 @@
     (u'WEAP',
         (u'"' + u'","'.join((_(u'Type'),_(u'Mod Name'),_(u'ObjectIndex'),
         _(u'Editor Id'),_(u'Weight'),_(u'Value'),_(u'Damage'),
-        _(u'Speed'),_(u'Reach'),_(u'EPoints'))) + u'"\n')),
+        _(u'Speed'),_(u'Reach'),_(u'EPoints'),_(u'Stagger'),
+        _(u'CritDamage'),_(u'criticalMultiplier'))) + u'"\n')),
```

Comment:
```python
    #--Weapons
    (u'WEAP',
        (u'"' + u'","'.join((_(u'Type'),_(u'Mod Name'),_(u'ObjectIndex'),
        _(u'Editor Id'),_(u'Weight'),_(u'Value'),_(u'Damage'),
        _(u'Speed'),_(u'Reach'),_(u'EPoints'),_(u'Stagger'),
        _(u'CritDamage'),_(u'criticalMultiplier'))) + u'"\n')),
```
So that it matches up with values in 
```python
#------------------------------------------------------------------------------
# StatsImporter
#------------------------------------------------------------------------------
statsTypes = {
        'ALCH':('eid', 'weight', 'value'),
        'AMMO':('eid', 'value', 'damage'),
        'APPA':('eid', 'weight', 'value'),
        'ARMO':('eid', 'weight', 'value', 'armorRating'),
        'BOOK':('eid', 'weight', 'value'),
        'INGR':('eid', 'weight', 'value'),
        'KEYM':('eid', 'weight', 'value'),
        'LIGH':('eid', 'weight', 'value', 'duration'),
        'MISC':('eid', 'weight', 'value'),
        'SLGM':('eid', 'weight', 'value'),
        'WEAP':('eid', 'weight', 'value', 'damage', 'speed', 'reach',
                'enchantPoints', 'stagger', 'critDamage','criticalMultiplier',
            ),
    }
```

Reply:
The order might need to be corrected though to match below
```python
        if bush.game.fsName in (u'Skyrim', u'Skyrim Special Edition') :
            self.attr_type = {'eid':self.sstr,	            self.attr_type = {'eid':self.sstr,
                              'weight':self.sfloat,	                              'weight':self.sfloat,
                              'value':self.sint,	                              'value':self.sint,
                               'damage':self.sint,	                               'damage':self.sint,
                               'armorRating':self.sint,	                               'armorRating':self.sint,
                               'duration':self.sint,	                               'duration':self.sint,
                               'speed':self.sfloat,	                               'speed':self.sfloat,
                               'reach':self.sfloat,	                               'reach':self.sfloat,
                              'stagger':self.sfloat,	                              'stagger':self.sfloat,
                              'enchantPoints':self.sint,	                              'enchantPoints':self.sint,
                              'critDamage':self.sint,	                              'critDamage':self.sint,
                              'criticalMultiplier':self.sfloat,	                              'criticalMultiplier':self.sfloat,}
```

Reply:
The results I was getting before the change
```
Imported Stats

•  Weapons: 2752

•  Cutting Room Floor.esp: 4
•  Dawnguard.esm: 100
•  Dragonborn.esm: 677
•  HearthFires.esm: 1
•  LegacyoftheDragonborn.esm: 296
•  Skyrim.esm: 1668
•  Weapons Armor Clothing & Clutter Fixes.esp: 6
```

Reply:
`Skyrim.esm: 1668` were all ITM

Reply:
I mean the newline

Reply:
Ah right, no idea, was always there I think.

Reply:
Check this should affect the patcher log (?) - re ITMs: I (still) don't get the patch process but could this somehow prevent a mod from carrying the criticalEffect to the patch?

Reply:
I'll have to test but I think removing it would prevent it being carried, but it does fall outside the scope of 'Stats' bash tag.